kodilla-spring/src/test/java/com/kodilla/spring/portfolio/BoardTestSuite.java

```diff
@@ -0,0 +1,37 @@
+package com.kodilla.spring.portfolio;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+@SpringBootTest
+public class BoardTestSuite {
+
+    @Test
+    void testSet() {
+        ApplicationContext context = new AnnotationConfigApplicationContext(BoardConfig.class);
+
+        boolean listToDoExist = context.containsBean("toDoList");
+        boolean listInProgressExist = context.containsBean("inProgressList");
+        boolean listDoneExist = context.containsBean("doneList");
+
+        System.out.println("List toDo exist: " + listToDoExist + " List inProgres exist: " + listInProgressExist + " List done exist: " + listDoneExist);
+    }
+
+    @Test
+    void addTest() {
+        ApplicationContext context = new AnnotationConfigApplicationContext("com.kodilla.spring.portfolio");
+        TaskList toDoList = (TaskList) context.getBean("toDoList");
+        TaskList inProgressList = (TaskList) context.getBean("inProgressList");
+        TaskList doneList = (TaskList) context.getBean("doneList");
+
+
+        toDoList.getTasks().add("newTasksListToDo");
+        System.out.println(toDoList);
+        inProgressList.getTasks().add("tasksListInProgress");
+        System.out.println(inProgressList);
+        doneList.getTasks().add("doneTasksList");
+        System.out.println(doneList);
```

Comment:
System.out.print w klasie testowej to nie jest dobry pomysl - trzeba zastosowac asercje